app/schema/api/current/models.rb

```diff
@@ -46,7 +46,8 @@ def patient
           age_updated_at: { '$ref' => '#/definitions/nullable_timestamp' },
           deleted_at: { '$ref' => '#/definitions/nullable_timestamp' },
           created_at: { '$ref' => '#/definitions/timestamp' },
-          updated_at: { '$ref' => '#/definitions/timestamp' } },
+          updated_at: { '$ref' => '#/definitions/timestamp' },
+          recorded_at: { '$ref' => '#/definitions/timestamp' } },
```

Comment:
Add it to required field in V3

Reply:
can't, because existing v3 apps are not sending it.